app/src/main/java/com/simplecity/amp_library/model/Album.java

```diff
@@ -166,8 +166,7 @@ public boolean equals(Object o) {
 
         Album album = (Album) o;
 
-        if (id != album.id) return false;
-        return name != null ? name.equals(album.name) : album.name == null;
+        return id == album.id && (name != null ? name.equals(album.name) : album.name == null);
```

Comment:
I'd prefer not to make this change. 'Simplifying' the conditional statement generally makes it harder to read for equals() method.